tempfile.go

```diff
@@ -133,6 +133,10 @@ func (t *PendingFile) CloseAtomicallyReplace() error {
 // going to write a large number of files to the same file system, store the
 // result of TempDir(filepath.Base(path)) and pass it instead of the empty
 // string.
+//
+// The file's permissions may vary from system to system (e.g. 0600 on macOS).
```

Comment:
As per https://github.com/golang/go/blob/41e62b8c49d21659b48a95216e3062032285250f/src/io/ioutil/tempfile.go#L65, this is not actually system-dependent.

So, let’s rephrase this to state outright that the file will have 0600 unless Chmod is called please :)

Reply:
Done.